include/base/dof_map.h

```diff
@@ -613,11 +609,7 @@ class DofMap : public ReferenceCountedObject<DofMap>,
    */
   unsigned int block_size() const
   {
-#ifdef LIBMESH_ENABLE_BLOCKED_STORAGE
     return (this->has_blocked_representation() ? this->n_variables() : 1);
```

Comment:
This looks like it's consistent either way, but without the preceding change it becomes effectively the same either way.

Reply:
Without this PR, I always have a block size of 1 regardless. I would like to libMesh to setup a correct block size for me, and I can use it for the matrix structure and the preconditioner.